erigon-lib/kv/membatchwithdb/memory_mutation_test.go

```diff
@@ -528,55 +528,37 @@ func TestAutoConversion(t *testing.T) {
 
 	initializeDbAutoConversion(rwTx)
 
-	batch := NewMemoryBatch(rwTx, "", log.Root())
```

Comment:
Can’t remove membatch from test of membatch. 

But i don’t remember - do we use membatch now in e3 or not (maybe not).